docs/pages/concepts/offloading/namespace-replication.md

```diff
@@ -0,0 +1,83 @@
+
+In the Liqo model, the namespace replication corresponds to a standard v1.Namespace and an associated [NamespaceOffloading](#) CR.
+The NamespaceOffloading resource allows to select the clusters which are the clusters where the application contained inside the namespace can be scheduled.
+
+In [a dedicated usage section](/usage/namespace_offloading), you can learn how to define and use a NamespaceOffloading resource activating the replication of the local namespace on peered clusters.
+
+In this section, you may discover how Liqo namespace model works under the hood, undestanding which resources and controllers are activated when a new namespaceOffloading is manipulated.
+
+#### Resources
+
+Liqo namespace replication involves instances of two kinds of CRDs:
+
+* First, the initial trigger is represented by the manipulation of a **NamespaceOffloading** (or the labelling of a namespace as explained in [the usage section](/usage/namespace_offloading)).
+  On the one hand, the *NamespaceOffloading* spec describes the properties of the replication, such as the name of the replicated namespaces.
+  On the other hand, the status collects the information about the actual status of remote namespaces (i.e. if the replication has succeeded).
+* Second, **NamespaceMap** resources contain the list of namespaces associated to a specific node.
+  The spec collects the list of desired namespaces for a specific remote cluster while the status updated information about their effective creation.
+  Each *NamespaceMap* can be alimented by several *NamespaceOffloading* instances that are targeting the same cluster for a remote namespace.
+  It is worth noting that the NamespaceMap status represent the source of truth to know the status of replicated namesapces on foreign clusters.
+
+#### Controllers
+
+The namespace replication logic is composed of several distinct controllers, responsible for different aspects of namespace controllers:
+
+* **NamespaceOffloading Controller**: processes the NamespaceOffloading spec and outputs namespace entries to NamespaceMap specs.
+* **NamespaceMap Controller**: enforces the namespace creation of the remote namespaces on remote clusters and updates the status of the NamespaceMaps.
+* **OffloadingStatus Controller**: updates the status of the NamespaceOffloading resources by reading the NamespaceMaps statuses.
+
+#### Workflow
+
+In the next figure, you can observe a representation of the overall workflow steps of the replication process:
+
+![](/images/namespace-replication/replication.png)
+
+We can resume the steps in:
+
+1. When the user creates/updates a **NamespaceOffloading** object in a Liqo-enabled namespace **(Step 1 in the figure)**, the Liqo logic processes the resource spec.
+
+2. After having detected the virtual-nodes compliant with selector, a namespace creation request is entered in every NamespaceMap associated with a selected cluster **(Step 2 in the figure)**.
```

Comment:
Please start this sentence with a subject. E.g.:
Component XX, after having bla bla, will do ble ble